plugin-server/src/worker/ingestion/process-event.ts

```diff
@@ -87,57 +87,16 @@ export class EventsProcessor {
                 throw new Error(`No team found with ID ${teamId}. Can't ingest event.`)
             }
 
-            if (data['event'] === '$snapshot') {
-                if (team.session_recording_opt_in) {
```

Comment:
I think I missed this in the change to the new recordings consumer 🤔

Reply:
please add a test too 😅

Reply:
Hah yup have added one for this 🙄